frontera/contrib/backends/remote/codecs/msgpack.py

```diff
@@ -30,7 +30,8 @@ def serialize(obj):
         elif hasattr(obj, '__dict__'):
             return serialize(obj.__dict__)
         else:
-            logger.warning('unable to serialize object: {}'.format(obj))
+            if obj:
```

Comment:
This logging problem is in fact a bug added by me in this [PR](https://github.com/scrapinghub/frontera/pull/233/files#diff-c2fba22f948d4da57c25c9c26263a9b7R33). Thank you for this! 

However your fix doesn't seem right to me. What happens when `obj=True` or say there is a key in a dict which has a value that is True? It will still output the warning message in such a case. 

In fact now If I think about this whole method looks buggy to me. Whenever a dict contains booleans which can be easily serialized this method will convert them into None. 

This method made more sense when it would encode string objects to `utf-8`. But this was removed [here](https://github.com/scrapinghub/frontera/pull/233/files#diff-c2fba22f948d4da57c25c9c26263a9b7R18) as part of the fix. 
What this method really does of value  now is that it [copies](https://github.com/scrapinghub/frontera/blob/master/frontera/contrib/backends/remote/codecs/msgpack.py#L22) the dict. But IMO, this was needed when strings were forced to be encoded to `utf-8`. I no longer feel that this method is needed. 
If you're really interested in fixing this than you can try removing this method completely and test it by adding some unit tests.